geany.desktop.in.in

```diff
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Type=Application
-Version=1.0
-_Name=Geany
+Version=@VERSION@
```

Comment:
This is not the app's version: http://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#key-version